package.json

```diff
@@ -17,11 +17,8 @@
     "check": "node scripts/check-codestyle.js",
     "prerelease": "node scripts/verdaccio.js",
     "release": "node scripts/release.js",
-    "prebuild:publish": "npm install && node scripts/prebuild.js --publish",
-    "build:publish": "npm run check && npm run build && npm run test ",
-    "postbuild:publish": "node scripts/postbuild.js",
     "prebuild:release": "node scripts/prebuild.js",
-    "build:release": "npm install && npm run check && npm run build && npm run test ",
+    "build:release": "npm run check && npm run build && npm run test ",
```

Comment:
How come the `install` is not needed anymore?

Reply:
since we use it in the script below, we thought itd be easier to seperate them, otherwise we'll always have to wait to npm install again every time we call this script